numpy/lib/function_base.py

```diff
@@ -3119,6 +3185,9 @@ def corrcoef(x, y=None, rowvar=True, bias=np._NoValue, ddof=np._NoValue):
         Has no effect, do not use.
 
         .. deprecated:: 1.10.0
+    covmat : array_like, optional 
+        A pre-computed covariance matrix of the variables. All other
+        arguments are ignored if `covmat` is provided.
```

Comment:
Having said that, I would prefer this API and no `cov2corr` function, to avoid polluting the namespace:

```
def corrcoef(x=np._NoValue, y=None, rowvar=True, cov=np._NoValue):
    if cov is np._NoValue != x is np._NoValue:
        raise TypeError("Exactly one of 'x' and 'cov' must be passed")
    if x is not np._NoValue:
        covmat = np.cov(x, y, rowvar)
    else:
        covmat = cov
    # everything else below
```